assets/training/finetune_acft_hf_nlp/components/pipeline_components/nlp_multiclass/spec.yaml

```diff
@@ -0,0 +1,332 @@
+
+  precision:
+    type: string
+    enum:
+    - '32'
+    - '16'
+    default: '16'
+    optional: true
+    description: Apply mixed precision training. This can reduce memory footprint
+      by performing operations in half-precision.
+
+  # MLFlow Parameters
+  enable_full_determinism:
+    type: string
+    enum:
+    - 'true'
+    - 'false'
+    default: 'false'
+    optional: true
+    description: Ensure reproducible behavior during distributed training
+
+  evaluation_strategy:
+    type: string
+    default: epoch
+    optional: true
+    enum:
+    - epoch
+    - steps
+    description: The evaluation strategy to adopt during training
+
+  evaluation_steps_interval:
+    type: number
+    default: 0.0
+    optional: true
+    description: The evaluation steps in fraction of an epoch steps to adopt during
+      training. Overwrites evaluation_steps if not 0.
+
+  evaluation_steps:
+    type: integer
+    default: 500
+    optional: true
+    description: Number of update steps between two evals if evaluation_strategy='steps'
+
+  logging_strategy:
+    type: string
+    default: steps
+    optional: true
+    enum:
+    - epoch
+    - steps
+    description: The logging strategy to adopt during training.
+
+  logging_steps:
+    type: integer
+    default: 500
+    optional: true
+    description: Number of update steps between two logs if logging_strategy='steps'
+
+  primary_metric:
+    type: string
+    default: accuracy
+    optional: true
+    enum:
+    - loss
+    - f1_macro
+    - mcc
+    - accuracy
+    - precision_macro
+    - recall_macro
+    description: Specify the metric to use to compare two different models
+
+  # Deepspeed Parameters
+  apply_deepspeed:
+    type: string
+    enum:
+    - 'true'
+    - 'false'
+    default: 'true'
+    optional: true
+    description: If set to true, will enable deepspeed for training
+
+  # ORT Parameters
+  apply_ort:
+    type: string
+    enum:
+    - 'true'
+    - 'false'
+    default: 'true'
+    optional: true
+    description: If set to true, will use the ONNXRunTime training
+
+  # MLFlow Parameters
+  save_as_mlflow_model:
+    type: string
+    enum:
+    - 'true'
+    - 'false'
+    default: 'true'
+    optional: true
+    description: If set to true, will save as mlflow model with pyfunc as flavour
+
+  batch_size:
+    type: integer
+    default: 32
+    optional: true
+    description: batch size for pre-process
+
+  deepspeed_config:
+    type: uri_file
+    optional: true
+    description: Deepspeed config to be used for finetuning
+
+outputs:
+  pytorch_model_folder_finetune:
+    type: uri_folder
+    description: Output dir to save the finetune model and other metadata
+
+  mlflow_model_folder_finetune:
+    type: mlflow_model
+    description: Output dir to save the finetune model as mlflow model
+
+jobs:
+  model_import:
+    type: command
+    component: azureml://registries/azureml/components/text_classification_model_import/versions/0.0.10
```

Comment:
Should this syntax be "azureml:text_classification_model_import:0.0.10", so that staging registry would pick up components from staging, preview from preview etc. ?

Reply:
Updated as suggested